### DISCUSSION TITLE
Can not install/configure Sonoff Zigbee Bridge Pro.

### DISCUSSION THREAD

**Discussion by deep414** [arendst/Tasmota#21989] [Support request] [answered]
Hi, Gurus.
I tried to install sonoffzigbee bridge pro s/w on my brand new Sonoff Zigbee Bridge Pro device.
steps I did:
1. successefully uploaded s/w to sonoff device.
2. connected it to my wifi network
3. connected to device via browser.
4. I saw, that Zigbee Permit Join grayed out. OK.
![zigbeepermit](https://github.com/user-attachments/assets/6892f28c-e3d6-4682-9ac7-df25963be2a1)
5. I looked into console and saw
![logform console](https://github.com/user-attachments/assets/efc89614-7da4-428b-9ab6-bae2c7ceecb1)
6. OK. I went to configuration -> auto-configuration and see that my current configuration Sonoff ZBPro.
![currentautoconfiguration](https://github.com/user-attachments/assets/5e1b509d-0614-4e52-a532-c0101e6a37de)
7. I went to configuration -> configure module
![moduleparameters](https://github.com/user-attachments/assets/d5350ba2-a4d7-439e-ba3b-af04877ad257)
I believe that my zigbee device did not started. But why? What did I miss? Can someone shed some light on this issue?

With best regards.

Alexei

**Comment by sfromis** [comment]
Did you also follow the instructions about flashing replacement firmware to the CC2652 Zigbee chip?

**Reply by deep414** [reply]
No. I did not updated firmware for CC2652 Zigbee chip. Will you be so kind as to point me to correct procedure?
Thanks.
Alexei

**Reply by sfromis** [reply, marked as answer]
https://zigbee.blakadder.com/Sonoff_ZBBridge-P.html

**Reply by deep414** [reply]
Thanks a lot.
Alexei
